platform/ios/src/MGLMapView.mm

```diff
@@ -3893,7 +3884,7 @@ - (MGLMapCamera *)cameraForCameraOptions:(const mbgl::CameraOptions &)cameraOpti
         return self.residualCamera;
     }
 
-    mbgl::CameraOptions mapCamera = self.mbglMap.getCameraOptions();
+    mbgl::CameraOptions mapCamera = self.mbglMap.getCameraOptions(cameraOptions.padding.value_or(mbgl::EdgeInsets()));
```

Comment:
I'm confused, could you please explain why are you passing the `cameraOptions.padding`

Reply:
no need for it, used it in debugging. cleaned.